packages/core/src/components/Modal/private/Inner.tsx

```diff
@@ -78,21 +81,23 @@ export class ModalInner extends React.Component<ModalInnerProps & WithStylesProp
 
   render() {
     const {
-      cx,
       children,
       footer,
       image,
       large,
       small,
       fluid,
       scrollable,
-      styles,
       subtitle,
       title,
       topBar,
       topBarCentered,
+      styleSheet,
+      innerContentStyleSheet,
     } = this.props;
 
+    const [styles, cx] = useStyles(styleSheet ?? styleSheetInner);
```

Comment:
you can't use a hook in a class component right?

Reply:
No you cannot.